tests/manager/manager-executeReadWriteCommand_error-001.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+MongoDB\Driver\Manager::executeReadWriteCommand()
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+
+$command = new MongoDB\Driver\Command([]);
+
+echo throws(function() use ($manager, $command) {
+	$manager->executeReadWriteCommand(
+		DATABASE_NAME, $command,
+		[
+			'readPreference' => new \MongoDB\Driver\ReadPreference(\MongoDB\Driver\ReadPreference::RP_SECONDARY),
```

Comment:
We're not in a namespace so the leading `\` isn't really necessary.

Reply:
It's a habit... do you want me to change them (everywhere)?